### PR TITLE
[Merged by Bors] - Add Hourly tests

### LINKED ISSUE #924: Run benchmark tests every hour
With #879 complete, we should be able to run some form of benchmark tests in CI, so we can begin to collect basic information about the performance of Fluvio.

We can start with running `producer_stress` with a local and k8 cluster.

Depends on #923

### PR DESCRIPTION
**Pull request by tjtelan**
Split off of #923 

Add new hourly test job which will run smoke and performance tests at the beginning of every hour.

Resolves #924

### CODE REVIEW

File: .github/workflows/hourly.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          kubernetes version: 'v1.19.2'
+      #- Install helm
+      - name: Install Helm
+        run: actions/ci-replace-helm.sh
+        env:
+          HELM_VERSION: v3.3.4
+          OS: ${{ matrix.os }}
+      #- Install and start Fluvio
+      - name: Install and start Fluvio cluster
+      #  - Matrix: local, k8
```
Comment (nicholastmosher) [inline .github/workflows/hourly.yml:54]: There is no `local` in the matrix

File: .github/workflows/hourly.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          SCCACHE_VERSION: 0.2.13
+        run: |
+          SCCACHE_FILE=sccache-$SCCACHE_VERSION-x86_64-unknown-linux-musl
+          mkdir -p $HOME/.local/bin
+          curl -L "$LINK/$SCCACHE_VERSION/$SCCACHE_FILE.tar.gz" | tar xz
+          mv -f $SCCACHE_FILE/sccache $HOME/.local/bin/sccache
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+      - name: Install sccache (macos-latest)
+        if: matrix.os == 'macos-latest'
+        run: brew install sccache
```
Comment (nicholastmosher) [inline .github/workflows/hourly.yml:71]: There was a brownout of brew the other day, I believe they want people to use the newest version (or newest repos?) on brew. Can you make this into

```
run: |
  brew update
  brew install sccache
```

**Comment by nicholastmosher** [comment]
Looks good. Can you do the same thing here where you just include changes for hourly on this PR and leave benchmark stuff in the other PR?

File: .github/workflows/hourly.yml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+        cluster_type: [k8]
+        test:
+          - name: smoke 
+            cmd: cargo run --release --bin flv-test -- smoke --develop --disable-install -- --producer-iteration 100 --producer-record-size 1000
```
Comment (sehz) [inline .github/workflows/hourly.yml:19]: can you put producer-iteration and producer recored size as env parameter in the job?

**Comment by tjtelan** [comment]
> Looks good. Can you do the same thing here where you just include changes for hourly on this PR and leave benchmark stuff in the other PR?

I need the benchmark stuff to merge first before this one. I am running the `producer_stress` test in benchmark mode.

**Review by sehz — CHANGES_REQUESTED** [review]
LGMT.  couple of minor nits

**Comment by sehz** [comment]
rebase

**Review by sehz — APPROVED** [review]
We can defer some cleanup in later PR

**Pull request closed without merging**